src/Resources/views/search_theme.html.twig

```diff
@@ -4,7 +4,20 @@
         <ul>
             <li>{{- form_widget(form) -}}{{ form_label(form) }}</li>
         </ul>
+    </div>
+{% endblock %}
 
+{% block setono_sylius_meilisearch_facet_range_row %}
+    <div class="facet facet-range">
+        {{ form_label(form) }}
+        <div class="ui small labeled input" style="margin-bottom: 1rem;">
```

Comment:
I actually didn't use any Semantic CSS markup because I reckon nobody uses it, so by not using it, maybe, just maybe they won't have to override this file :D

You don't have to fix it. I can just fix it in another PR :)